statick_tool/tool_plugin.py

```diff
@@ -18,7 +18,7 @@ class ToolPlugin(IPlugin):  # type: ignore
 
     plugin_context = None
 
-    def get_name(self) -> str:
+    def get_name(self) -> str:  # type: ignore[empty-body]
```

Comment:
I'm not sure suppression on these is right.  I first tried the `mypy` suggestion of returning a value, but that caused failures in the unit testing.   My guess is that the unit testing depends on returning `None` in some cases, rather than the string suggested by the typing?  Feel free to suggest a more extensive fix if this isn't right.

Reply:
I'm not sure why code coverage drops with this PR.